share/commands/server-trigger-policy-generation

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# @description trigger a policy generation
+# @man This command will trigger a policy generation, to generate
+# @man policies for any nodes that may have changed
+# @man +
+# @man *Options*:
+# @man +
+# @man *-i*: run the agent in information mode, displays all executed commands
+# @man +
+# @man *-c*: run the agent without color output
+
+. "${BASEDIR}/../lib/common.sh"
+. "${BASEDIR}/../lib/api_call.sh"
+
+TOKEN="$(cat /var/rudder/run/api-token)"
+DISPLAY_COMMAND=false
+
+while getopts "iIvdc" opt; do
+  case $opt in
+    i|I|v|d)
+      DISPLAY_COMMAND=true
+      ;;
+    c)
+      clear_colors
+      ;;
+  esac
+done
+
+# check if the API is available
+api_result=$(filtered_api_call "${API_URL}/api/system/status" "${TOKEN}" "GET" -f)
```

Comment:
This is only available in 5.0+, and the request need an api version, most likely "latest"